src/AngleSharp.Css/Values/Primitives/Constant2.cs

```diff
@@ -0,0 +1,80 @@
+namespace AngleSharp.Css.Values
```

Comment:
Personally, I'd prefer having both definitions in the same file.

Alternatively, we could rename the first file (`Constant.ts`) to `Constant1.cs`.